app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Improving work history
+description: Asking for a full work history and simplifying the section by removing skills and experience.
+date: 2021-01-08
+related:
+  items:
+  - text: Work history breaks
+    href: https://bat-design-history.netlify.app/apply-for-teacher-training/work-history-breaks/
+    description: Previous post about breaks in work history
+  - text: Work history - August iteration
+    href: https://bat-design-history.netlify.app/apply-for-teacher-training/work-history/
+    description: Previous change to the work history flow
+---
+Many providers told us that they want to see a candidate's full work history, not just the last 5 years - for safeguarding reasons.
+
+Candidates were spending a significant amount of time filling in the “Skills and experience relevant to teaching” question for each job, treating these as a “mini personal statement”. However, providers indicated that they didn’t always read these in detail.
+
+Analysis of validation errors also revealed that candidates were often leaving the “Skills and experience relevant to teaching” question blank. Our hypothesis was that they wanted to come back and write this section later.
+
+## What we did
+
+We moved the work history section beneath the “Safeguarding” heading on the application review page. This was to try and communicate to candidates that the purpose of the work history was mainly for safeguarding, rather than something used to directly assess them.
+
+We changed the “How long have you been working?” question to “Do you have any work history?”. We didn't need to guide candidates towards entering only their last 5 years of work history anymore.
+
+We added an answer to this question – “No, I have always been in full time education”. This was based on analysis showing that around 20% of candidates are 20 or 21 years old, and have likely gone straight from A-Levels to University, which may explain their lack of work history. Adding this option means that they don't have to explain this in their own words.
+
+If someone selected “No, I do not have any work history”, we revealed a textbox asking “Tell us why you’ve been out of the workplace” with some hint text which gave three examples. These were very brief, to indicate to candidates that they did not need to be super detailed.
```

Comment:
I am not sure why you're saying this? Did you change this in any way?

Reply:
Yes, previously the textbox was on a follow-up page rather than conditionally-revealed.

Reply:
And the hints were new. I'll see if I can make this clearer.